base/dump-dotnet6/aws-lambda-tools-defaults.json

```diff
@@ -0,0 +1,8 @@
+﻿{
+  "configuration": "Release",
+  "framework": "net6.0",
+  "function-runtime": "dotnet6.0",
```

Comment:
Should runtime name be `dotnet6` - this is how AWS has it?

Reply:
Good point - I've updated.